AGENTS.md

```
@@ -0,0 +1,143 @@
+```
+JSON (tests/fixtures/*.json)
+  → serde → SceneSpec (builder/scene.rs)
+  → build_scene() → Vec<Box<dyn RiveObject>>
+  → encode_riv() → Vec<u8>
+  → .riv file
+  → validate_riv() / inspect_riv() (validator/mod.rs)
+```
+
+## DEPENDENCY GRAPH
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)